web/src/engine/websites/MangaPill.ts

```diff
@@ -0,0 +1,30 @@
+import { Tags } from '../Tags';
+import icon from './MangaPill.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+
+let pathMangas = '/search?page={page}&';
+const genres = ['Action', 'Adventure', 'Cars', 'Comedy', 'Dementia', 'Demons', 'Doujinshi', 'Drama', 'Ecchi', 'Fantasy', 'Game', 'Gender Bender', 'Harem', 'Hentai'];
```

Comment:
May use multiline array initialiser instead of composing with push
```javascript
const genres = [
    'a', 'b', 'c',
    'd', 'e', 'f',
    // ...
];
```